drape/support_manager.cpp

```diff
@@ -101,27 +99,32 @@ void SupportManager::ForbidVulkan()
   settings::Set(kVulkanForbidden, true);
 }
 
-bool SupportManager::IsVulkanForbidden() const
+bool SupportManager::IsVulkanForbidden()
 {
   bool forbidden;
   if (!settings::Get(kVulkanForbidden, forbidden))
     forbidden = false;
   return forbidden;
 }
 
-bool SupportManager::IsVulkanForbidden(std::string const & deviceName,
-                                       Version apiVersion, Version driverVersion) const
+bool SupportManager::IsVulkanForbidden(std::string const & deviceName, Version apiVersion, Version driverVersion)
 {
-  static char constexpr * kBannedDevices[] = {
+  static char constexpr const * kBannedDevices[] = {
```

Comment:
Well, constexpr adds only buzzword here.
Probably, you wanted to write something like ```char const * constexpr kBannedDevices``` ... But anyway